code/datums/quirks/negative.dm

```diff
@@ -1006,77 +1003,3 @@
 	lose_text = span_notice("You feel a growing strength in your vocal chords.")
 	medical_record_text = "The patient is unable to use their voice in any capacity."
 	hardcore_value = 4
-
-/datum/quirk/body_purist
```

Comment:
why body purist removal?

Reply:
It was not merged in skyrat, but somehow made it into my changes, as far as I can tell!

Reply:
See here for the actual Body Purist PR that hasnt been merged yet https://github.com/Skyrat-SS13/Skyrat-tg/pull/17472